docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -289,14 +286,251 @@ paths:
+
+            * `*foo.com`
+            * `u_n:*`
+            * `file://*`
+            * `http://foo.com*`
+
+            <mark>This feature is experimental and the API may change.</mark>
+          schema:
+            type: string
+        - name: user
+          in: query
+          example: acct:username@hypothes.is
+          description: Limit the results to annotations made by the specified user.
+          schema:
+            type: string
+            pattern: "acct:^[A-Z0-9._]{3,30}@.*$"
+        - name: group
+          in: query
+          example: "8JmD3iz1"
+          description: Limit the results to annotations made in the specified group (by group ID).
+          schema:
+            type: string
+        - name: tag
+          in: query
+          example: "artificial intelligence"
+          description: |
+            Limit the results to annotations tagged with the specified value.
+
+            For example: `artificial intelligence` will find all annotations whose tags
+            contain both `artificial` **AND** `intelligence`.
+          schema:
+            type: string
+        - name: tags
+          in: query
+          example:
+            - artificial
+            - intelligence
+          description: |
+            Similar to `tag` but allows a comma-separated list of multiple tags.
+
+            For example: `[intelligence,artificial]` will find all annotations whose tags
+            contain both `artificial` **AND** `intelligence`.
+          schema:
+            type: array
+            items:
+              type:
+                string
+        - name: any
+          in: query
+          example: "ribosome"
+          description: |
+            Limit the results to annotations who contain the indicated keyword
+            in any of the following fields:
+
+            * `quote`
+            * `tags`
+            * `text`
+            * `url`
+          schema:
+            type: string
+        - name: quote
+          in: query
+          example: "unicorn helmets"
+          description: >
+            <p>Limit the results to annotations that contain this text inside
+            the text that was annotated.</p>
+
+            <p>For example: `unicorn helmets` would return all annotations containing
+            `unicorn` **OR** `helmets` in their quoted (i.e. annotated) text.</p>
+          schema:
+            type: string
+        - name: references
+          in: query
+          description: Returns annotations that are replies to this parent annotation ID.
+          schema:
+            type: string
+        - name: text
+          in: query
+          example: "penguin strength"
+          description: >
+            <p>Limit the results to annotations that contain this text in their
+            textual body.</p>
+
+            <p>For example: `penguin strength` would return all annotations containing
+            `penguin` **OR** `strength` in their text (body) content.</p>
```

Comment:
If you were to search for these terms independently (such as: text:penguin, text:strength) they would match in an AND like behavior but since they are one term with a space they match in OR like behavior. @lyzadanger and I went through and updated this documentation for search ~ 3 months back and we haven't made any search changes since then so these should all be correct still.